internal/cli/log_streams.go

```diff
@@ -159,7 +48,7 @@ func logStreamsCmd(cli *cli) *cobra.Command {
 	cmd.SetUsageTemplate(resourceUsageTemplate())
 	cmd.AddCommand(listLogStreamsCmd(cli))
 	cmd.AddCommand(showLogStreamCmd(cli))
-	cmd.AddCommand(createLogStreamCmd(cli))
+	cmd.AddCommand(createLogStreamsCmd(cli))
```

Comment:
```suggestion
	cmd.AddCommand(createLogStreamCmd(cli))
```

Reply:
Fixed in [`b509205` (#599)](https://github.com/auth0/auth0-cli/pull/599/commits/b509205955d9015aa54a79350ec6f17c1db76b96)